misc/hp-omnibook-x14-fe0xxx/badcode.asl

```diff
@@ -0,0 +1,406 @@
+/*
+ * badcode.asl
+ *
+ * This file contains examples of the extended error checking and
+ * typechecking capabilities of the iASL compiler. Other ASL compilers
+ * may ignore these errors completely. Note - this is not an exhaustive
+ * list of errors detected by iASL, it shows many of the errors that
+ * are not detected by other ASL compilers.
+ *
+ * To compile, use:
+ * iasl badcode.asl
+ *
+ * Output:
+ * Compilation complete. 45 Errors, 28 Warnings, 11 Remarks, 14 Optimizations
+ *
+ */
+DefinitionBlock ("badcode.aml", "DSDT", 1, "Intel", "Example", 0x00000001)

```

Comment:
Most likely this AML shouldn't be a part of this patch. Could you please clarify it?

Reply:
Oh I wasn't sure. Will remove it, no prob